source/_references/10-operations.md

```diff
@@ -167,14 +167,18 @@ Converts an image to grayscale.
 ### Modulate
 
 Modulates an image for Brightness, Saturation and Hue. Use this to for example brighten up or darken an image or 
-remove its colors. The value is meant as percentage, with which the different values are multiplied. 
-100 leaves the property untouched, 0 removes all of it. 
+remove its colors. For brightness and saturation, the value is meant as percentage, with which the different values are multiplied. 
+100 leaves the property untouched, 0 removes all of it.
```

Comment:
re #80 we could add here that "200 duplicates the brightness or saturation respectively".

Reply:
I changed the whole text, hopefully clearer now